contrib/confluentinc/confluent-kafka-go/kafka/kafka.go

```diff
@@ -108,6 +124,11 @@ func (c *Consumer) startSpan(msg *kafka.Message) ddtrace.Span {
 		tracer.Tag(ext.MessagingSystem, "kafka"),
 		tracer.Measured(),
 	}
+
+	if bs, err := c.kafkaConfig.Get("bootstrap.servers", ""); err == nil {
```

Comment:
Is this safe to do? Any user who is using the deprecated `WrapConsumer` directly will have a `nil` kafkaConfig if I'm reading this correctly

Reply:
Extracted the value into separate struct upon first wrapping so this issue is resolved. There is now a check to confirm if the value is stored in the struct first (which is only possible in `WrapConsumerWithConfig`)